flink-python/pyflink/table/table_environment.py

```diff
@@ -1664,11 +1663,13 @@ def create(stream_execution_environment: StreamExecutionEnvironment = None,  # t
             # create with StreamExecutionEnvironment.
             >>> env = StreamExecutionEnvironment.get_execution_environment()
             >>> table_env = StreamTableEnvironment.create(env)
-            # create with StreamExecutionEnvironment and TableConfig.
-            >>> table_config = TableConfig()
-            >>> table_env = StreamTableEnvironment.create(env, table_config)
             # create with StreamExecutionEnvironment and EnvironmentSettings.
-            >>> environment_settings = EnvironmentSettings.in_streaming_mode()
+            >>> configuration = Configuration()
+            >>> configuration.set_string('execution.buffer-timeout', '1 min')
+            >>> environment_settings = EnvironmentSettings
+            >>>     .new_instance()
+            >>>     .in_streaming_mode()
+            >>>     .with_configuration(configuration)
```

Comment:
is there no `build()` method at the end?

Reply:
Apologies @twalthr, I've rushed. fixed it.